src/autoonline.cpp

```diff
@@ -91,6 +75,44 @@ static bool IsPoeRunning() {
 #endif
 }
 
+bool is_poe_running_remotely(const std::string& script) {
+#if defined(Q_OS_LINUX) || defined(Q_OS_MAC)
+    std::string poe_check_script = "/bin/sh -c \"" + script + "|grep PathOfExile|grep -v grep|wc -l\"";
```

Comment:
Wouldn't it be better if this just executed the script, without grep'ing anything? The script should just do its thing and exit with, let's say, code 0 if poe is running and code 1 if it isn't.